### PR TITLE
Instruction tracing

### PR DESCRIPTION
**Pull request by lvb2000**

### CODE REVIEW

File: ext/orbetto/meson.build [hunk 1]
```diff
@@ -21,11 +21,17 @@ liborb = orbuculum.get_variable('liborb')
 orbinc = orbuculum.get_variable('incdirs')
 git_version_info_h = orbuculum.get_variable('git_version_info_h')
 
+
+# Specify the include directory for Roaring Bitmap
+cpp = meson.get_compiler('cpp')
+roaring_inc = include_directories(['subprojects/CRoaring/build'])
+roaring_lib = cpp.find_library('libcroaring-singleheader-source-lib', dirs: ['/Users/lukasvonbriel/Auterion/embedded-debug-tools/ext/orbetto/subprojects/CRoaring/build'])
```
Comment (lvb2000) [inline ext/orbetto/meson.build:28]: This is going to be fixed later

Comment (lvb2000) [reply ext/orbetto/meson.build]: Done

File: ext/orbetto/src/mortrall.hpp [hunk 2]
```diff
@@ -0,0 +1,1220 @@
...
+                            }
+                            _stackReport(Mortrall::r);
+                        }
+                        else
+                        {
+                            /* The branch wasn't taken, so just move along */
+                            Mortrall::r->op.workingAddr += ( ic & LE_IC_4BYTE ) ? 4 : 2;
+                        }
+                    }
+                    else if (ic & LE_IC_SYNC_BARRIER)
```
Comment (niklaut) [inline ext/orbetto/src/mortrall.hpp:571]: optimize if else

File: ext/orbetto/subprojects/orbuculum.wrap
Comment (niklaut) [inline ext/orbetto/subprojects/orbuculum.wrap:1]: Also check and upstream orbculum changes.
https://github.com/niklaut/orbuculum/commit/ac9514e14e0b748b750cc06af7d2ca8302ff8519#diff-f3899d23668ce3cd983412d4a617caa657c6dbb5ada1766a546ef37d6d9b56ebR57

**Review by niklaut — COMMENTED** [review]
See also required patches in https://github.com/Auterion/PX4_firmware_private/commit/033975139a65c65f4d129659baf0a66d96ea88d3

**Pull request merged**